src/gapp_login/wechat/migrations/versions/450921ce3e56_wechat_login_initial.py

```diff
@@ -0,0 +1,26 @@
+"""wechat login initial
+
+Revision ID: 450921ce3e56
+Revises:
+Create Date: 2020-06-17 15:18:58.686238
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "450921ce3e56"
+down_revision = None
+branch_labels = ("wechat",)
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "CREATE INDEX identities_wechat_unionid_idx "
+        "    ON identities (CAST(profile ->> 'wechat_unionid' AS CHARACTER VARYING)) "
```

Comment:
I met an problem here, now I try to update `indentities` table in gapp-login package. If I have a empty database and run `poetry run alembic upgrade heads`, it will be failed with `relation "identities" does not exist`. But it works if I already upgraded everything before I run this verison.
Do you have any idea?

Reply:
https://alembic.sqlalchemy.org/en/latest/branches.html#branch-dependencies this should solve the problem

Reply:
fixed